bot/cogs/tags.py

```diff
@@ -27,6 +26,44 @@ def __init__(self, bot: Bot):
         self.bot = bot
         self.tag_cooldowns = {}
 
+        self._cache = {}
+        self._last_fetch = None
+
+    async def _get_tags(self, is_forced: bool = False) -> None:
+        """Getting all tags."""
+        # Refresh only when there's a more than 5m gap from last call.
+        if is_forced or not self._last_fetch or time.time() - self._last_fetch > 5 * 60:
+            tags = await self.bot.api_client.get('bot/tags')
+            self._cache = {tag['title'].lower(): tag for tag in tags}
+
+    @staticmethod
+    def _fuzzy_search(search: str, target: str) -> bool:
```

Comment:
```suggestion
    def _fuzzy_search(search: str, target: str) -> int:
```

Reply:
Thank you, I will include this in a single commit with the suggestion below.

Reply:
While you're at it, I believe there are a couple of commented out lines that you could probably just delete entirely.